core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -974,6 +1002,41 @@ private ExprValue exprTimeToSec(ExprValue time) {
     return new ExprLongValue(time.timeValue().toSecondOfDay());
   }
 
+  /**
+   * UTC_DATE implementation for ExprValue.
+   * Returns in format `yyyy-MM-dd`.
+   *
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcDate() {
+    return new ExprDateValue(exprUtcTimeStamp().dateValue());
+
+  }
+
+  /**
+   * UTC_TIME implementation for ExprValue.
+   * Returns in format `HH:mm:ss`.
+   *
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTime() {
+    return new ExprTimeValue(exprUtcTimeStamp().timeValue());
+  }
+
+  /**
+   * UTC_TIMESTAMP implementation for ExprValue.
+   * Returns in format `yyyy-MM-dd HH:mm:ss`.
+   *
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTimeStamp() {
+    String formattedLDT = LocalDateTime.now().format(DATE_TIME_FORMATTER_NON_STRUCT);
+    ExprStringValue localDateValue = new ExprStringValue(formattedLDT);
+    ExprStringValue toTZ = new ExprStringValue("+00:00");
+
+    return exprDateTime(localDateValue, toTZ);
```

Comment:
Is this supposed to return a Datetime or Timestamp?

Reply:
Timestamp is formatted as a datetime. We have no `exprTimeStamp`.

Reply:
Would https://github.com/Bit-Quill/opensearch-project-sql/blob/bc91c66b572a564221ea87b7f126f2158e0ac083/core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java#L987 work?